### ISSUE TITLE
I have to re-import shell sessions every time

### ISSUE THREAD

**Issue by allegrox** [kingToolbox/WindTerm#2119]
Every time I start Windterm, I have to re-import shell sessions, re-import them or skip them, and the next time I have to re-import them.

**Comment by allegrox** [comment]
Found a configuration item in the config file that has solved the problem.
Suggest a document, some features are not sure what they do.

**Comment by gllrmzndm** [comment]
I have the same problem, where did you find it? I'm on Debian 12 GNOME (Linux)

**Comment by allegrox** [comment]
> I have the same problem, where did you find it? I'm on Debian 12 GNOME (Linux)

in file /opt/WindTerm/global/wind.config
set "terminal.reimportShellSessions": false

**Comment by damosse31** [comment]
Duplicate #2100

**Comment by CasparLi** [comment]
In my macOS system, 
change file "~/.wind/profiles/default.v10/user.config" not work.
modify the wind.config file in folder  "/Applications/WindTerm.app/Contents/MacOS/global" resolved.

**Comment by BrianHolsen** [comment]
for me, it to change this option to 2 in ~/.wind/profiles/default.v10/user.config
{
   "terminal.reimportShellSessions" : 2
}

**Comment by kingToolbox** [comment]
Yes, I encountered the same issue during usage as well. I will fix it. Please stay tuned, thank you.

**Comment by tv21** [comment]
Just for the benefit of anyone who may come across this, in version 2.6.1 (at least in the Linux version; the MacOS version may be different) the location of this setting appears to have changed back to what allegrox reported, it is now in <program directory>/global/wind.config and the setting you are looking for is

"terminal.reimportShellSessions": true,

and you want to change the true to false.  Do NOT try to add the setting to ~/.wind/profiles/default.v10/user.config if it's not there or you will lose all your configuration settings.  Don't ask me how I know! :(

(To the developer:  How about a "Don't ask again" checkbox on that popup?)

**Comment by kingToolbox** [comment]
[WindTerm 2.7.0 Prerelease 2](https://github.com/kingToolbox/WindTerm/releases/tag/2.7-prerelease-2) has addressed this issue. The Windows and Linux versions are available for download now. Please download and try it. Thank you.

**Comment by bj600800** [comment]
In my case, the config file location is /Applications/WindTerm.app/Contents/MacOS/global/wind.config
press "/" and input terminal.reimportShellSessions, change true to false.
Problem solved.
